CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Added
```

Comment:
Is all of this appropriate to go under the `Added` heading? There's also a [`Changed`](https://keepachangelog.com/en/1.1.0/#how) heading that could be used.